pkg/minikube/image/image.go

```diff
@@ -104,8 +105,24 @@ func WriteImageToDaemon(img string) error {
 	if err != nil {
 		return errors.Wrap(err, "getting tag")
 	}
+	glog.V(3).Infof("Writing image %v", tag)
 	_, err = daemon.Write(tag, i)
-	return err
+	if err != nil {
+		return errors.Wrap(err, "writing image")
+	}
+
+	//TODO: Make pkg/v1/daemon accept Ref too
+	//      Only added it to pkg/v1/tarball
+
+	glog.V(3).Infof("Pulling image %v", ref)
+
+	// Pull digest
+	cmd = exec.Command("docker", "pull", "--quiet", img)
```

Comment:
use podman, if the user's driver is podman

Reply:
same as code in line 87, cmd := exec.Command("docker", "images", "--format", "{{.Repository}}:{{.Tag}}@{{.Digest}}")

they both should use podman I believe, if user has podman as the driver.right?

Reply:
I didn't change that part, it was hardcoded to "docker" already before

Reply:
yeah it was wrong before this PR.

Reply:
This will need to implement WriteImageToDaemon, so that it works with podman.